guides/v2.1/cloud/deploy/static-content-deployment.md

```diff
@@ -0,0 +1,61 @@
+
+By default, the [STATIC_CONTENT_SYMLINK environment variable]({{page.baseurl}}/cloud/env/variables-deploy.html#static_content_symlink) is set to `true`. After generating the static content during the build phase, it creates a symlink to the content folder.
+
+Generating static content requires access to themes and locales. Magento stores themes in the file system, which is accessible during the build phase; however, Magento stores locales in the database. The database is _not_ available during the build phase. In order to generate the static content during the build phase, you must move locales to the file system. The {{site.data.var.ct}} package contains the `config:dump` command that reads locales and saves them in the `app/etc/config.php` file.
+
+1.  Log in to your Cloud environment using SSH and move locales to the file system, then update the [`config.php` file]({{site.baseurl}}/guides/v2.2/cloud/project/project-upgrade.html#create-a-new-configphp-file).
+
+1.  The `.magento.env.yaml` configuration file should contain the following values:
+
+    -  [SKIP_HTML_MINIFICATION]({{page.baseurl}}/cloud/env/variables-global.html#skip_html_minification) is set to `true`
+    -  [SKIP_SCD]({{page.baseurl}}/cloud/env/variables-build.html#skip_scd) on build stage is set to `false`
+    -  [SCD_STRATEGY]({{site.baseurl}}/guides/v2.2/cloud/env/variables-build.html#scd_strategy) is set to `compact`
+
+1.  Verify configuration of the [Post-deploy hook]({{page.baseurl}}/cloud/project/project-conf-files_magento-app.html#hooks) in the `.magento.app.yaml` file.
+
+1.  Verify your settings by running the [Smart wizard for the ideal state]({{page.baseurl}}/cloud/deploy/smart-wizards.html).
+
+    ```bash
+    php ./vendor/bin/ece-tools wizard:ideal-state
+    ```
+
+### Setting the SCD on demand
+
+Generating SCD on demand is optimal for a development workflow. It decreases deployment time so that you can quickly review your implementations and run integration tests. Enable the [SCD_ON_DEMAND]({{page.baseurl}}/cloud/env/variables-global.html#scd_on_demand) environment variable in the global stage of the `.magento.env.yaml` file. The SCD_ON_DEMAND variable overrides all other configurations related to SCD and clears existing content from the `~/pub/static` directory.
+
+When using the SCD on-demand strategy, it helps to preload the cache with pages you expect to request, such as the home page. Add your list of expected pages in the [WARM_UP_PAGES]({{page.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) environment variable in the post-deploy stage of the `.magento.env.yaml` file.
+
+### Skipping SCD
+
+In some cases you could choose to skip generatating static content completely. You can set the [SKIP_SCD]({{page.baseurl}}/cloud/env/variables-build.html#skip_scd) environment variable in the global stage to ignore other configurations related to SCD. This does not affect existing content in the `~/pub/static` directory.
```

Comment:
Spelling: generatating

Reply:
OMG, that did NOT show in the spell checker. I need to fix the dictionary. grrrr...